app/javascript/components/modals/TaskHintsModal.tsx

```diff
@@ -14,20 +14,30 @@ export const TaskHintsModal = ({
   onClose: () => void
 }): JSX.Element => {
   return (
-    <Modal open={open} onClose={onClose} className="m-editor-hints" {...props}>
-      <header>
-        <GraphicalIcon icon="hints" category="graphics" />
-        <h2>Hints and Tips</h2>
-      </header>
-      <div className="single-task-hints">
-        <h3>{task.title}</h3>
-        {task.hints.map((hint, idx) => (
-          <div
-            className="c-textual-content --large"
-            key={idx}
-            dangerouslySetInnerHTML={{ __html: hint }}
-          ></div>
-        ))}
+    <Modal
+      open={open}
+      onClose={onClose}
+      aria={{
+        describedby: 'task-hints-description',
+      }}
+      className="m-editor-hints"
+      {...props}
+    >
+      <div id="task-hints-description" role="document">
```

Comment:
```suggestion
      <div id="a11y-task-hints-description" role="document">
```

Let's explicitly prefix these ids with `a11y` so that we understand that's what they're for and don't at some point in the future accidentally remove them, or refactor them out